test/config-next/nonce-b.json

```diff
@@ -0,0 +1,27 @@
+{
+    "NonceService": {
+        "maxUsed": 131072,
+        "syslog": {
+            "stdoutLevel": 6
+        },
+        "debugAddr": ":8112",
+        "grpc": {
+            "address": ":9102",
```

Comment:
I think you probably don't need to split the nonce config JSON into a and b. What we do for other services is:

(a) use the commandline override `--debug-addr` in startservers.py to avoid debugaddr conflicts, and
(b) bind to different IP addresses to avoid conflicts on the service port.

Note that you'll still have each nonce service including itself as a backend, which is slightly inefficient but I think not too terrible.

Reply:
Ah yup, good idea. I'm not a huge fan of having the circular backend stuff, but eh, for the simplification I think it's probably worth it.

Reply:
Yeah, and keep in mind the same will be true in prod because of the way we do service discovery for DNS. Our main option to avoid it would probably be to define a custom gRPC balancer again that takes the A records from DNS and filters out the IP address we bound our own service to. Right now we're using an off-the-shelf balancer.